### PR TITLE
added tests for CVSB-11504

### PR DESCRIPTION
**Pull request by DeepikaSinghD**

### CODE REVIEW

File: src/main/resources/loader/develop/test-results_put_payload_11504.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    "numberOfWheelsDriven": null,
+    "odometerReading": 123,
+    "odometerReadingUnits": "kilometres",
+    "preparerId": "QE7355",
+    "preparerName": "RPT Logistics Limited",
+    "reasonForCancellation": null,
+    "regnDate": "2011-01-05",
+    "systemNumber": "KC7VFFUPB5HYLXTG",
+    "testEndTimestamp": "2020-04-08T04:22:27.612Z",
+    "testerEmailAddress": "cvs.automation2@dvsagov.onmicrosoft.com",
```
Comment (dpanzaru-deloitte) [inline src/main/resources/loader/develop/test-results_put_payload_11504.json:20]: Tanio said not to use real email addresses any more in the json files we add in the project, so we should change these

Comment (DeepikaSinghD) [reply src/main/resources/loader/develop/test-results_put_payload_11504.json]: change done

File: src/main/java/clients/TestResultsClient.java [hunk 2]
```diff
@@ -520,6 +520,30 @@ public Response postVehicleTestResultsWithAlterations(String body, List<JsonPath
         return response;
     }
 
+    public Response putTestResultsWithAlterations(String vin, String requestBody, List<JsonPathAlteration> alterations) {
+        Response response = callPutTestResultsWithAlterations(vin, requestBody, alterations);
+
+        if (response.getStatusCode() == 401 || response.getStatusCode() == 403) {
```
[the remark this answers is not in the document]
Comment (DeepikaSinghD) [reply src/main/java/clients/TestResultsClient.java]: Changes done.

File: src/test/java/testresults/PutTestResults.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        // Collate all alterations
+        List<JsonPathAlteration> alterations = new ArrayList<>(Arrays.asList(
+                alterationSystemNumber,
+                alterationVin,
+                alterationTestResultId,
+                restriction));
+
+        //Put the request payload with alterations
+        testResultsSteps.putTestResultsWithAlterations(randomSystemNumber,putRequestBody,alterations);
+        testResultsSteps.statusCodeShouldBe(400);
```
[the remark this answers is not in the document]
Comment (DeepikaSinghD) [reply src/test/java/testresults/PutTestResults.java]: Changes done.

**Pull request merged**